spec/descriptions/tagSyntheticResults.md

```diff
@@ -205,6 +214,10 @@ Either tagFilters or tagFilterExpression can specify a custom property by its ke
       "name": "synthetic.locationId", 
       "operator": "EQUALS", 
       "stringValue": "abcdefgXSJmQsehOWg1S"
+    }, {
+      "name": "synthetic.testActive",
+      "operator": "EQUALS",
+      "booleanValue": true
```

Comment:
synthetic.testActive is only applicable to getTestSummaryList API call, so this change and the change above need to go to the section: Get a list of Synthetic tests with Success Rate and Average Response Time data.